jekyll/_cci2/server/overview/circleci-server-v4-overview.adoc

```diff
@@ -200,40 +195,59 @@ but can also be configured to be exported to other places (such as Datadog or Pr
 | No jobs will be sent to Nomad. The run queue will increase in size, but there should be no meaningful loss of data.
 |
 
+| distributor-*
+| App Core
+| Responsible for accepting build requests and destributing the job to appropriate queues.
+|
+|
+
 | domain-service
 | App Core
 | Stores and provides information about our domain model. Works with permissions and API.
 | Workflows will fail to start and some REST API calls may fail, causing 500 errors in the CircleCI UI. If LDAP authentication is in use, all logins will fail.
 |
 
-| exim
-|
-| Will be removed in GA, but users can provide mail submission credentials to an existing MTA.
-| No email notifications will be sent.
-|
-
 | frontend
 | Frontend
 | CircleCI web app and www-api proxy.
 | The UI and REST API will be unavailable and no jobs will be triggered by GitHub/Enterprise. Running builds will be OK, but no updates will be seen.
 | Rate limit of 150 requests per second with a single user instantaneous limit of 300 requests. 
 
-| inject-bottoken
+| insights-service
+| Metrics
+| A service to aggregate build and usage metrics for exporting and analysis.
+| 
 |
-| A Kubernetes job that inserts a "bot token" into MongoDB. Bot tokens are authorization interservice communication.		
+
+| kong
+| App Core
+| API management.
+| 
 |
-| Mainly for www-api
+
 
 | legacy-notifier
 | App Core
 | Handles notifications to external services (for example, Slack or email).
 |
 |
 
-| prometheus
-| Server
-| Used for metrics.
+| nginx
+| App Core / Frontend
+| Handles traffic redirection and ingress.
+| 
 |
+
+| nomad-autoscaler
+| Nomad
+| Manages scaling of Nomad clusters in AWS and GCP environments.
+| 
+|
+
+| nomad-server
+| Nomad
+| On-cluster nomad server reponsible for managing off-cluster nomad clients.
```

Comment:
NIT:

'On/off-cluster' might be confusing here because we are now dealing with k8s and nomad clusters.  I might vote to simplify it, something like:

```suggestion
| Responsible for managing nomad clients.
```